contracts/rbn-staking/GaugeController.vy

```diff
@@ -558,16 +558,19 @@ def vote_for_gauge_weights(_gauge_addr: address, _user_weight: uint256):
     log VoteForGauge(block.timestamp, msg.sender, _gauge_addr, _user_weight)
 
 @external
-def disperse_funds(_total_funding: uint256):
+def disperse_funds(_total_funding: uint256, _duration: uint256 = WEEK):
```

Comment:
people can change their weights for a particular gauge once [every 10 days](https://github.com/ribbon-finance/governance/blob/bc4b0867b00e765b47daa2d8b34b0440070e14c1/contracts/rbn-staking/GaugeController.vy#L16) in the crv implementation

Reply:
10 days sounds fine